api/bootstrap.proto

```diff
@@ -208,8 +210,17 @@ message Runtime {
 
 // Rate limit :ref:`configuration overview <config_rate_limit_service>`.
 message RateLimitServiceConfig {
-  // Specifies the cluster manager cluster name that hosts the rate limit
-  // service. The client will connect to this cluster when it needs to make rate
-  // limit service requests.
-  string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
+  oneof service_specifier {
+    option (validate.required) = true;
+
+    // Specifies the cluster manager cluster name that hosts the rate limit
+    // service. The client will connect to this cluster when it needs to make rate
+    // limit service requests.
```

Comment:
Is it worth documenting the difference between the two choices in the one-of?  Maybe not, because this one is deprecated.  But which gRPC client will get used in the deprecated case?  Or is it even the same protocol between these two?

Reply:
Will fix.